README.md

```
@@ -0,0 +1,143 @@
+##### 16. Add module-info.java which :
+
+    - use interfaces
+    - requires module with Bank implementation
+    - requires module with Service implementation
+    - requires jmp.dto;
+
+##### 17. Demonstrate functionality of created classes.
+
+Tasks 18-20 weight: 40 points
+
+##### 18. Add a new default method for Service interface, which uses getAllUsers. Use LocalDate.now(), ChronoUnit and mapToLong.
```

Markdownlint

Punctuation: '.'
Trailing punctuation in heading

(MD026, no-trailing-punctuation)